frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-dashboards.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Helmet } from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+
+import {
+  useExtensions,
+  DashboardsCard,
+  DashboardsTab,
+  isDashboardsCard,
+  isDashboardsTab,
+} from '@console/plugin-sdk';
+import {
+  getPluginTabPages,
+  mapStateToProps,
+  DashboardsPageProps,
+} from '@console/internal/components/dashboard/dashboards-page/dashboards';
+import { HorizontalNav, PageHeading, LoadingBox } from '@console/internal/components/utils';
+
+const OCSDashboardsPage: React.FC<DashboardsPageProps> = ({ match, kindsInFlight, k8sModels }) => {
+  const { t } = useTranslation();
+  const title = t('ceph-storage-plugin~OpenShift Container Storage Overview');
+  const tabExtensions = useExtensions<DashboardsTab>(isDashboardsTab);
+  const cardExtensions = useExtensions<DashboardsCard>(isDashboardsCard);
```

Comment:
You cannot use these extensions. They are meant for Cluster overview. If anybody will want to add a new tab next tu cluster overview it will appear here.